### DISCUSSION TITLE
Docker help

### DISCUSSION THREAD

**Discussion by black-jonathan** [community/community#53311] [Programming Help]
### Body

I have a docker container with python app in it. I am using docker compose up, but I added one package and now I can't access terminal, because it crashes everytime I start it up. I tried deleting the container, but it didn't help. Can anyone help?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by matej-kaska** [comment]
You can try to delete your container and it's image. Then it should rebuild itself and work.

**Reply by black-jonathan** [reply]
I didn't know, that you need to delete image too, thank you!

**Reply by queenofcorgis** [reply]
@black-jonathan @matej-kaska GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
